components/playlist/browser/playlist_background_web_contentses.cc

```diff
@@ -32,8 +32,8 @@ const char* GetUserAgentOverride(const GURL& url) {
     return kUserAgentOverride;
   }
 
-  static const auto kSites = base::MakeFixedFlatSet<net::SchemefulSite>(
-      base::sorted_unique, {net::SchemefulSite(GURL("https://ted.com"))});
+  static const base::flat_set<net::SchemefulSite> kSites = {
```

Comment:
`base::NoDestructor`

Reply:
Fixed in [6a9e985](https://github.com/brave/brave-core/pull/22320/commits/6a9e98567947495cd9127da25a3033c08054c009)